packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -400,7 +400,9 @@ export default class QuanticCategoryFacet extends LightningElement {
   }
 
   get canShowLess() {
-    return this.facet && this.state?.canShowLessValues;
+    return (
+      this.facet && this.state?.canShowLessValues && !this.isFacetSearchActive
```

Comment:
this is not to collapse when facet search is enabled?

Reply:
This is not about the collapse button, it's about the the show less button that allow you show less facet values.

Reply:
ahhhh ok nevermind :P